eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -67,9 +58,9 @@ def clinvar_to_evidence_strings(dir_out, allowed_clinical_significance=None, ign
     record_counter = 0
     n_total_clinvar_records = 0
     skip = 0
-    limit = BATCH_SIZE
+    limit = config.BATCH_SIZE
```

Comment:
Usages of the `limit` variable could be replaced by direct access to `config.BATCH_SIZE` and would be aligned with the usages of `config.HOST` in the same lines.